### ISSUE TITLE
IE 11 flex issue

### ISSUE THREAD

**Issue by kyra-diablo** [zippyui/react-datagrid#75]
I had an isse that the datagrid did not had a height in IE 11. 
The issue came from the index-no-normalize.css, that for the inner content of the datagrid (.react-datagrid > z.inner) flex: 1 was specified, which is not always the same in all browser. (see https://stackoverflow.com/questions/21600345/flexbox-and-internet-explorer-11-displayflex-in-html)

I have a quick fix for now:
.react-datagrid > .z-inner {
    -ms-flex: 1 0 auto !important;
    flex: 1 0 auto !important;
}

**Comment by radubrehar** [comment]
Yes, it should be set to `1 0 auto`. Will release an update with this. Thanks for pointing that out

**Comment by rjeffer** [comment]
Do you know why react datagrid doesnt show in IE or Safari but works fine in Chrome and Firefox?
